ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -212,6 +215,33 @@ public enum CocoaPodUtils {
     }
   }
 
+  /// Get all transitive pod dependencies for a pod.
+  /// - Returns: An array of Strings
+  static func transitivePodDependencies(for podName: String,
+                                        in installedPods: [String: PodInfo]) -> [String] {
+    var newDeps = Set([podName])
+    var returnDeps = Set<String>()
+    repeat {
+      var foundDeps = Set<String>()
+      for dep in newDeps {
+        let childDeps = installedPods[dep]?.dependencies ?? []
+        foundDeps.formUnion(Set(childDeps))
+      }
+      newDeps = foundDeps.subtracting(returnDeps)
+      returnDeps.formUnion(newDeps)
+    } while newDeps.count > 0
+    return Array(returnDeps)
+  }
+
+  /// Get all transitive pod dependencies for a pod.
+  /// - Returns: An array of VersionedPod
```

Comment:
Nit: An array of dependencies with versions for a given pod.

Reply:
Done.